app/views/static_pages/kontakt.html.erb

```diff
@@ -1,23 +1,45 @@
 <% provide(:title, "Kontakt") %>
-<section class="spa-area spa-area-no-space" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html">
-  <div class="widget spa-widget-owl-slide-home1">
-    <div class="owl-carousel spa-owl-slider">
-      <article class="entry-item">
-        <div class="entry-thumb">
-          <%= image_tag ('3.jpg') %>
+
+<section class="spa-area spa-area-17">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="widget spa-widget-portfolio">
+          <div class="widget-header style-08">
```

Comment:
this structure from section till this line appears 4 times here, maby let's etract it to layout?